app/templates/gentelella/guest/event/cfs.html

```diff
@@ -100,7 +100,7 @@ <h4 class="modal-title">{{ _("Add Speaker Information") }} </h4>
                                                 <h4>{{ _("Existing Sessions") }} </h4>
                                                 <p>No Existing Sessions! Please Click on <b>Add Session Proposal</b> to submit a new session.<br><br></p>
                                             </div>
-                                            <a href="{{ url_for('my_sessions.process_speaker_view',speaker=user_speaker[0]['id']) }}" type="button" class="btn btn-info" id = "speaker-button">{{ _("Edit Speaker Details") }}</a>
+                                            <a href="{{ url_for('my_sessions.process_speaker_view',speaker_id=user_speaker[0]['id']) }}" type="button" class="btn btn-info" id = "speaker-button">{{ _("Edit Speaker Details") }}</a>
```

Comment:
Why is this change part of this PR ? This was done in #3570. This shouldn't come up in the diff. Please fix your commits.